messenger.rst

```diff
@@ -209,6 +220,35 @@ This will also configure the following services for you:
 
         $ composer require symfony/amqp-pack
 
+Redis
+~~~~~
+
+The redis transport will use the redis streams to queue messages.
+
+.. code-block:: bash
+
+    # .env
+    ###> symfony/messenger ###
+    MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages
+    ###< symfony/messenger ###
+
+This is enough to allow you to route your message to the ``redis`` transport.
+
+If you have multiple systems to receive the same messages you could use different groups
+to achieve this:
+
+.. code-block:: bash
+
+    # .env
+    ###> symfony/messenger ###
+    MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages/group1/consumer1
+    ###< symfony/messenger ###
+
+.. note::
+
+    In order to use Symfony's built-in Redis transport, you will need the Redis
```

Comment:
Can we add the required `composer` command here so lazy people like me can just copy&paste? :P

Reply:
Is this not system specific how the extension is installed? It could be `pecl install redis` or `apt-get` or something else or?

Reply:
Ah I see, misunderstood!